quickstarts/kubernetes/kubernetes-log-ingest-analysis/config.yml

```diff
@@ -0,0 +1,54 @@
+id: 88a79f52-81c1-4fa2-99f8-3cb5b012f160
```

Comment:
Since this is a new quickstart, we will generate this `id` once it has been merged.
```suggestion
```